src/app.py

```diff
@@ -22,10 +26,15 @@
 DATABASE = os.getenv("DATABASE")
 VOLUME_NOTIFICATIONS_ENDPOINT = os.getenv("VOLUME_NOTIFICATIONS_ENDPOINT")
 GOOGLE_SERVICE_ACCOUNT_PATH = os.getenv("GOOGLE_SERVICE_ACCOUNT_PATH")
+SERVER = os.getenv("SERVER")
 
+if SERVER == "prod":
```

Comment:
why is it that the GOOGLE_SHEET_IDs are in the codebase and not in an environments file? could they be abused?